docs-new/docs/contribute/general/contrib-guidelines-gen.md

```diff
@@ -43,3 +43,12 @@ please create an issue on the GitHub repository.
 * If you want to do the work on an issue,
   include that information in your description of the issue
   or in a comment to the issue.
+
+## Proposing New Work
+
+* Create and Refine a Ticket
+  * When proposing new work, start by creating an issue or ticket in the project's issue tracker.
+  * Actively participate in [refinement meetings](https://community.cncf.io/keptn-community/)
```

Comment:
```suggestion
  * Actively participate in the refinement meetings sessions that are part of the weekly
  [community meetings(https://community.cncf.io/keptn-community/).
  In these sessions, everyone discusses the proposed work,
  whether it is a good idea, what exactly should be done.
```
Whoops, I kind of botched this vis a vis the next phrase.  But this is just a suggestion that we say a little more about what the refinement process is.  Do we also want to explain the "Scrum Poker" where, after the discussion, the maintainers vote about the size of this work and decide whether or not it should be done.

Unless your issue is a crucial and obvious bug, work on an issue should not commence until the issue has been refined.

You can straighten out the prose ;-)